.github/CODEOWNERS

```diff
@@ -1,3 +1,6 @@
 * @cognitedata/developer-tooling
 
 /packages/playground/src/api/documents @cognitedata/grep
+/packages/playground/src/__tests__/api/documents @cognitedata/grep
+/packages/stable/src/api/documents @cognitedata/grep
+/packages/stable/src/__tests__/api/documents @cognitedata/grep
```

Comment:
This does not match the tests files. Maybe it's easily solved by added a `*` at the end of those?

Reply:
Ah, yes, true. It isn't a documents folder, it's a file matching `documents*`

Reply:
I have updated this now